### ISSUE TITLE
Arrows on Itinerary are not useful

### ISSUE THREAD

**Issue by kira0204** [devil-Coder/itc#12]
Hey,
Arrows on the `itinerary` page is not useful, it is useful in the cases where the list is long for ex. [this](https://in.pycon.org/2018/#journey) on mobile. 
If possible, reuse those arrows to move around different days.

**Comment by hhk998402** [comment]
@devil-Coder  Please check this out

**Comment by devil-Coder** [comment]
Yes in the newly updated website it may useful as I have increased the distance.
@kira0204 Please review the changes, if required I will remove it.
Also please specify the screen for which it is not useful as in small screens or large ones?

**Comment by kira0204** [comment]
I raised the issue on PC though, will check for mobile and revert
